db/model/api.model.js

```diff
@@ -16,3 +16,18 @@ exports.selectArticleById = (article_id) => {
     return rows[0];
   });
 };
+
+exports.selectArticles = () => {
+  const chosenColumns =
+    "articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url";
+
+  const commentCount = "COUNT(comments.comment_id)::int AS comment_count";
+
+  return db
```

Comment:
As this query is unlikely to change, you could just do this whole query as one string without using the template literal! :)